docs/services/matrix.md

```diff
@@ -108,4 +108,16 @@ PGPASSWORD=S3cr3T \
   WHERE name='@test:test.com';"
 ```
 
+### Federation 
+
+To enable cross-server communication you need to set an SRV DNS record.
+
+```
+;; SRV Records
+_matrix._tcp.matrix.domain.de.    1    IN    SRV    10 5 443 matrix.domain.de.
+```
+
+![](../img/dns/dns-srv.png)
```

Comment:
- Enable lazy loading:
- Put Image into `services/` directory and call it `matrix-dns.jpg`
```
![DNS configuration](../img/services/matrix-dns.jpg){: loading=lazy }
```